pkg/kubernetes/client/client.go

```diff
@@ -117,15 +125,29 @@ func getDefaultLabels(pv string) map[string]string {
 // CreateJivaVolume check whether JivaVolume CR already exists and creates one
 // if it doesn't exist.
 func (cl *Client) CreateJivaVolume(req *csi.CreateVolumeRequest) error {
+	var sizeBytes int64
 	name := req.GetName()
+	name = strings.ToLower(name)
+	// CR only support names upto 63 chars
+	// so this trims the rest of the trailing chars
+	if len(name) > maxNameLen {
+		name = name[:maxNameLen]
+	}
```

Comment:
Please add this check in DeleteVolume req also

Reply:
Thanks for pointing this out